Classic Momentum.Rmd

```diff
@@ -0,0 +1,70 @@
+---
+title: "Classic Momentum"
+subtitle: "How to smack things together"
+author: "Kevin Rozmiarek"
+date: "Last knitted `r format(Sys.Date(), '%d %b %Y')`"
+output:
+  html_document: 
+    df_print: paged
+    number_sections: yes
+    toc: yes
+    toc_float: true
+    toc_depth: 3
+    code_folding: show
+editor_options:
+  chunk_output_type: inline
+---
+
+# Learning Goals
+
+1. Understand that momentum is conserved in collisions.
+
+2. Understand the difference between **elastic** and **inelastic** collisions
+
+3. Be able to balence and compute the momentum on objects.
+
+# Background
+
+Momentum is a quality of everything! It is the quantity of motion of a moving body, measured as a product of its mass and velocity. Momentum by itself isn't all that interesting, only when it comes to collisions! Let's talk about two different kinds of collisions first.
+
+**Elastic collision:** An elastic collision is an encounter between two bodies in which the total momentum and kinetic energy of the two bodies remains the same. In an ideal, perfectly elastic collision, there is no net conversion of kinetic energy into other forms such as heat, noise, or potential energy.
+
+**Inelastic Collision:** An inelastic collision, in contrast to an elastic collision, is a collision in which momentum and kinetic energy is not conserved due to the action of internal friction.
```

Comment:
same with internal friction - could say just a couple words defining what that is

Reply:
yup